marklogic-data-hub-central/ui/e2e/cypress/support/components/model/property-table.tsx

```diff
@@ -41,6 +41,10 @@ class PropertyTable {
   getPiiIcon(propertyName: string) {
     return cy.get(`[data-testid=pii-${propertyName}]`);
   }
+
+  getAddPropertyToStructureType(structureTypeName: string) {
+    return cy.get(`[data-testid=add-struct-${structureTypeName}]`);
```

Comment:
For future, if we are using data-testid, then try using cy.findByTestId
Our framework now imports  custom Cypress commands from RTL.

Reply:
Got it. I can do some refactoring the next time I do a commit.